.circleci/config.yml

```diff
@@ -206,7 +213,30 @@ jobs:
           working_directory: ~/expo/android
           command: nix-shell --run ./make-release.sh
       - save_gradle_cache
+      - save_cache:
+          key: client-android-apk-{{ .Revision }}
+          paths:
+            - ~/expo/android/exponent-release.apk
       - store_artifacts:
           path: ~/expo/android/exponent-release.apk
       - store_artifacts: # daemon logs for debugging crashes
           path: ~/.gradle/daemon/4.4
+
+  client_android_release_google_play:
+    executor: js
+    steps:
+      - setup
+      - yarn_install:
+          working_directory: ~/expo/tools
+      - yarn_install:
+          working_directory: ~/expo/tools-public
```

Comment:
If the gulp script is in `tools`, I don't think you need to install anything in `tools-public`.

Reply:
the gulp script step failed when I didn't do this, saying it couldn't find `@expo/json-file` required by `tools-public/project-versions.js`. I couldn't see any obvious dependency on this from `tools` so this was the easiest fix.